cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -3897,6 +3996,114 @@ func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeDegradedUnencrypted
 	c.Assert(filepath.Join(dirs.SnapBootstrapRunDir, fmt.Sprintf("%s-model-measured", s.sysLabel)), testutil.FilePresent)
 }
 
+func (s *initramfsMountsSuite) TestInitramfsMountsRecoverModeUnencryptedDataUnencryptedSaveHappy(c *C) {
+	// test a scenario when data is encrypted, same goes for save and the
+	// test observes calls to secboot unlock helper
```

Comment:
isn't this test setup the same as the two above already which I mentioned are duplicate?

Reply:
I think not, this one has unencrypted data and save, and both are mounted. Might get confused with an earlier test that tried unencrypted data, but encrypted save, in which case the save gets completely ignored.